gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -494,6 +469,41 @@ local key_params = {
 	walkkey = Key("+walk", "WALK")
 }
 
+-- handle finding Entities with a ray
+function FindEntityAlongView(pos, dir, filter)
```

Comment:
Takes position and direction as vectors, as well as a filter which entities should be shown. Therefore you can see yourself if you don't filter yourself and it doesnt change the way TargetIDs get drawn for everyone else without special addons.